docs/specifications/4-standard-compliance-suite.md

```diff
@@ -0,0 +1,166 @@
+# Standard Compliance Suite
+
+## Format
+In order to assert whether a new runtime implementation is correct, a formal
+compliance test suite is defined, that all new language runtimes must fully
+implement before they can be deemed eligible for General Availability.
+
+This document describes these tests, as well as a general approach for ensuring
+conformance requirements are met in a systematic manner.
+
+### Categories
+Test cases in the standard compliance suite are grouped by categories, which
+help implementors direct their effort in the early stages of the implementation
+of new language bindings. Each category is declared in an `H3` title (a line
```

Comment:
I understand your desire to be precise, but all the talk about how the document should be marked up feels like a waste of space to me. Write some examples and people will naturally start to copy/paste.

Are you planning to automatically process this MarkDown file using tools? If so, then that is something worthwhile to call out in a separate section, and you can specify the markup requirements there. Otherwise, I wouldn't bother.

Reply:
I am not sure I understand what categories are? Who defines them? Why do we need them?

Reply:
I imagine it will be thinks like "values", "callbacks", "async", "interfaces", "structs", etc.

Reply:
I am definitely planning on lifting data from the Markdown document by ways of a tool.